### ISSUE TITLE
RFC 2231 -vs- multiple attachments

### ISSUE THREAD

**Issue by coldsmile** [roundcube/roundcubemail#8308]
Issue appeared after upgrading from v1.4.10 to v1.5.0.

Having `$config['mime_param_folding'] = 0;` causes second/subsequent non-ascii attachment names to reset to `%messagepart%_%mime_id%.%mime_extension%`. The first attachment is not affected.

**Comment by alecpl** [comment]
I'm not sure I follow. Are we talking about message displaying or sending? Could you provide a sample? Does it work with all plugins disabled? Maybe a duplicate of #8282.

**Comment by coldsmile** [comment]
Disabling plugins doesn't help.
Filenames are displayed correctly after uploading. You could reveal the problem only after saving/sending...

[non-ascii first.eml.txt](https://github.com/roundcube/roundcubemail/files/7521874/non-ascii.first.eml.txt)
[non-ascii second.eml.txt](https://github.com/roundcube/roundcubemail/files/7521876/non-ascii.second.eml.txt)

**Comment by coldsmile** [comment]
fix from #8282 helped.
